lib/ts/recipe/session/sessionFunctions.ts

```diff
@@ -244,6 +247,13 @@ export async function getSessionInformation(
     recipeImplementation: RecipeImplementation,
     sessionHandle: string
 ): Promise<SessionInformation> {
+    let querier = Querier.getNewInstanceOrThrowError(SuperTokens.getInstanceOrThrowError().isInServerlessEnv);
```

Comment:
Please see how other functions in this file gets the `querier` -> `await recipeImplementation.querier`

Reply:
Done